browser-sync/browser-sync.d.ts

```diff
@@ -334,6 +334,8 @@ declare module "browser-sync" {
              * @param name the identifier used for retrieval
              */
             get(name: string): BrowserSyncInstance;
+
+            instances: BrowserSyncInstance[];
```

Comment:
I've looked, but I can't find any reference to this in the documentation.  Is this a case where BrowserSync's documentation needs to be updated, or is this a value used for private/internal use only?  In the former case, you can log an issue or pull request here: https://github.com/Browsersync/browser-sync/issues.  In the latter case, we probably wouldn't want it to be part of the type definition file.